src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,175 @@
+"""The unpolarized LO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+
+from eko import constants
+
+from ....harmonics import w1
+
+# from ....harmonics import cache as c
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, cache, is_singlet=None):
+    r"""Compute the LO quark-quark anomalous dimension.
+
+    Implements Eqn. (B.3) from :cite:`Mitov:2006wy`.
```

Comment:
```suggestion
    Implements :eqref:`B.3` from :cite:`Mitov:2006wy`.
```
see:
https://github.com/NNPDF/eko/blob/13197aa354f19779e5783b35437b8f7f780714ae/doc/source/shared/roles.rst?plain=1#L76-L77
https://github.com/NNPDF/eko/blob/13197aa354f19779e5783b35437b8f7f780714ae/src/eko/msbar_masses.py#L223